### ISSUE TITLE
Batch Norm missing?

### ISSUE THREAD

**Issue by ghost** [carpedm20/simulated-unsupervised-tensorflow#10]
Hey, just wondering if there was a reason that batch norm was left out of the resnet blocks. Thanks for the repo!

**Comment by carpedm20** [comment]
Did paper mention about batch normalization? I couldn't find it in the paper.

**Comment by ghost** [comment]
No, couldn't find it but I thought that was a common critical piece of a resnet block. I'm pretty new to this, been using/learning your dcgan code. You'd probably know better than I would.

**Comment by carpedm20** [comment]
Batch notmalization is not a critical piece of resnet as far as I know (but I don't much know about common residual blocks). Also this model is not that deep enough to consider it as a common residual network. DCGAN use batch norm but not residual blocks.

**Comment by ghost** [comment]
Gotcha, wanted to throw it out there just in case. Saw this paper that does super resolution similar to the simgan paper (feed an image in, instead of noise) https://arxiv.org/pdf/1609.04802.pdf, deeper network though, thought it was worth a mention too. Closing

**Comment by carpedm20** [comment]
I see. Thanks for the sharing 👍
